docs/products/postgresql/howto/setup-logical-replication.rst

```diff
@@ -0,0 +1,174 @@
+
+     SELECT subdbid, subname, subowner, subenabled, subslotname
+     FROM aiven_extras.pg_list_all_subscriptions();
+
+      subdbid |      subname      | subowner | subenabled | subslotname
+     ---------+-------------------+----------+------------+-------------
+        16401 | dest_subscription |       10 | t          | slot
+     (1 row)
+
+9. Verify the subscription status::
+
+    SELECT * FROM pg_stat_subscription;
+
+     subid |      subname      | pid | relid | received_lsn |      last_msg_send_time       |     last_msg_receipt_time     | latest_end_lsn |        latest_end_time
+    -------+-------------------+-----+-------+--------------+-------------------------------+-------------------------------+----------------+-------------------------------
+     16444 | dest_subscription | 869 |       | 0/C002360    | 2021-06-25 12:06:59.570865+00 | 2021-06-25 12:06:59.571295+00 | 0/C002360      | 2021-06-25 12:06:59.570865+00
+    (1 row)
+
+10. Verify the data is correctly copied over the Aiven for PostgreSQL target tables
+
+
+Remove unused replication setup
+-------------------------------
+
+It is important to remove unused replication setups, since the underlying replication slots in PostgreSQL forces the server to keep all the data needed to replicate since the publication creation time. If the data stream has no readers, there will be an ever-growing amount of data on disk until it becomes full.
+
+To remove an unused subscription, essentially stopping the replication, run the following command in the Aiven for PostgreSQL target database::
+
+    SELECT * FROM aiven_extras.pg_drop_subscription('dest_subscription');
+
+
+Verify the replication removal with::
+
+    SELECT * FROM aiven_extras.pg_list_all_subscriptions();
+
+     subdbid | subname | subowner | subenabled | subconninfo | subslotname | subsynccommit | subpublications
+    ---------+---------+----------+------------+-------------+-------------+---------------+-----------------
+    (0 rows)
+
+
+Manage inactive or lagging replication slots
+--------------------------------------------
+
+Inactive or lagging replication could cause problems in a database, like an ever increasing disk usage not associated to any growth of the amount of data in the database. Fulfilling the disk causes the database instance to stop serving clients thus a loss of service.
+
+1. Assess the replication slots status via ``psql``::
+
+    SELECT slot_name,restart_lsn FROM pg_replication_slots;
+
+The command output is like::
+
+        slot_name   │ restart_lsn
+     ───────────────┼─────────────
+      pghoard_local │ 6E/16000000
+           debezium | 5B/8B0
+     (2 rows)
+
+2. Compare the ``restart_lsn`` values between the replication slot in analysis (``debezium`` in the above example) and ``pghoard_local``: the hexadecimal difference between the them states how many write-ahead-logging (WAL) entries are waiting for the target ``debezium`` connector to catch up.
```

Comment:
I'm really confused. We didn't mention Debezium before, is it just an example here?

Reply:
I removed the debezium reference and added the `dest_slot` naming in line with the example above